packages/product-feed-google/src/Migrations/Version20230405111441.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ProductFeed\GoogleBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20230405111441 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $seoRobotsContentSettingCount = $this->sql(
+            'SELECT COUNT(*) FROM setting_values WHERE name = \'mailWhitelist\' AND domain_id = 1;'
+        )->fetchOne();
+
+        if ($seoRobotsContentSettingCount <= 0) {
+            $this->sql(
```

Comment:
This migration should be multidomain (MultidomainTrait). Otherwise, it's possible the emails will be sent from the second and other domains.

Reply:
Fixed.